src/components/swap/UnsupportedCurrencyFooter.test.tsx

```diff
@@ -0,0 +1,77 @@
+import userEvent from '@testing-library/user-event'
+import { Token } from '@uniswap/sdk-core'
+import { useUnsupportedTokens } from 'hooks/Tokens'
+import { mocked } from 'test-utils/mocked'
+import { fireEvent, render, screen, waitForElementToBeRemoved, within } from 'test-utils/render'
+import { getExplorerLink } from 'utils/getExplorerLink'
+
+import UnsupportedCurrencyFooter from './UnsupportedCurrencyFooter'
+
+const unsupportedTokenAddress = '0x4e83b6287588a96321B2661c5E041845fF7814af'
+const unsupportedTokenSymbol = 'ALTDOM-MAR2021'
+const unsupportedToken = new Token(1, unsupportedTokenAddress, 18, 'ALTDOM-MAR2021')
+const unsupportedTokenExplorerLink = 'www.blahblah.com'
+
+jest.mock('@web3-react/core', () => {
+  const web3React = jest.requireActual('@web3-react/core')
+  return {
+    ...web3React,
+    useWeb3React: () => ({
+      chainId: 1,
+    }),
+  }
+})
+
+jest.mock('../../hooks/Tokens')
+
+jest.mock('../../utils/getExplorerLink')
+
+describe('UnsupportedCurrencyFooter.tsx', () => {
+  it('matches base snapshot', () => {
+    mocked(useUnsupportedTokens).mockImplementation(() => ({ unsupportedTokenAddress: unsupportedToken }))
+    const { asFragment } = render(<UnsupportedCurrencyFooter show={true} currencies={[unsupportedToken]} />)
+    expect(asFragment()).toMatchSnapshot()
```

Comment:
nit: you could one-line this if you wanted to.

Reply:
keeping it the same for conventions sake (its done like this throughout all the unit tests atm i believe)